politics.html

```diff
@@ -18,13 +18,230 @@
     <main>
         <!-- ADD THE MAIN SECTION HERE -->
 
+   <!-- section heading -->
+   <span class="section-heading">
+    <div class="left"></div>
+    <h2 class="section-heading-text">Politics</h2>
+    <div class="right"></div>
+</span>
 
-    </main>
-    <footer>
-        <!-- ADD THE FOOTER SECTION HERE -->
-
+<!-- main news -->
+<section class="main-news">
+    <div class="first-card">
+        <a href="https://punchng.com/apc-convention-omisore-12-others-make-south-west-unity-list/">
+            <img src="./img/Omisore.webp" alt="APC convention">
+            <p class="main-news-text2">APC convention: Omisore, 12 others make South-West unity list
+            </p>
+        </a>
         
-    </footer>
```

Comment:
Please remove this footer section. We don't need more than one footer. Please give it another name.